rotkehlchen/tests/unit/decoders/accounting_settings/test_default_settings.py

```diff
@@ -0,0 +1,338 @@
+        sequence_index=0,
+        timestamp=TIMESTAMP_1_MS,
+        location=Location.BLOCKCHAIN,
+        location_label=EXAMPLE_ADDRESS,
+        asset=A_ETH,
+        balance=Balance(amount=ONE),
+        notes='Received 1 ETH',
+        event_type=event_type,
+        event_subtype=event_subtype,
+    )
+    consumed_num = transactions.process(
+        event=eth_gain_event,
+        events_iterator=iter([]),
+    )
+    assert consumed_num == 1
+
+
+def test_accounting_no_settings(accounting_pot: 'AccountingPot'):
+    """Test that if there are no settings provided, the event is not taken into account"""
+    event = HistoryBaseEntry(
+        event_identifier=EXAMPLE_EVM_HASH,
+        sequence_index=0,
+        timestamp=TimestampMS(0),
+        location=Location.BLOCKCHAIN,
+        event_type=HistoryEventType.INFORMATIONAL,
+        event_subtype=HistoryEventSubType.NONE,
+        asset=A_ETH,
+        balance=Balance(),
+    )
+    was_skipped = False
+
+    def log_mock(text):
+        assert 'Skipping...' in text
+        nonlocal was_skipped
+        was_skipped = True
+
+    with patch('rotkehlchen.accounting.transactions.log.debug', side_effect=log_mock):
+        consumed_num = accounting_pot.transactions.process(event=event, events_iterator=iter([]))
+
+    assert was_skipped is True and consumed_num == 1
+
+
+@pytest.mark.parametrize('event_type,event_subtype,is_taxable,airdrops_taxable', [
+    (HistoryEventType.RECEIVE, HistoryEventSubType.NONE, True, False),
+    (HistoryEventType.WITHDRAWAL, HistoryEventSubType.NONE, False, False),
+    (HistoryEventType.RECEIVE, HistoryEventSubType.AIRDROP, False, False),
+    (HistoryEventType.RECEIVE, HistoryEventSubType.AIRDROP, True, True),
+    (HistoryEventType.RECEIVE, HistoryEventSubType.REWARD, True, False),
+])
+@pytest.mark.parametrize('mocked_price_queries', [prices])
+def test_accounting_receipts_settings(
+        accounting_pot: 'AccountingPot',
+        event_type: 'HistoryEventType',
+        event_subtype: 'HistoryEventSubType',
+        is_taxable: bool,
+):
+    """Test that the default accounting settings for recepts are correct"""
+    _gain_one_ether(
+        transactions=accounting_pot.transactions,
+        event_type=event_type,
+        event_subtype=event_subtype,
+    )
+
+    expected_event = ProcessedAccountingEvent(
+        type=AccountingEventType.TRANSACTION_EVENT,
+        notes='Received 1 ETH',
+        location=Location.BLOCKCHAIN,
+        timestamp=TIMESTAMP_1_SECS,
+        asset=A_ETH,
+        free_amount=ZERO if is_taxable else ONE,
+        taxable_amount=ONE if is_taxable else ZERO,
+        price=Price(FVal(1862.06)),
+        pnl=PNL(),
+        cost_basis=None,
+        index=0,
+        extra_data={'tx_hash': EXAMPLE_TX_HASH_HEX},
+    )
+    expected_event.calculate_pnl(
+        count_entire_amount_spend=False,
+        count_cost_basis_pnl=is_taxable,
+    )
+    assert accounting_pot.processed_events == [expected_event]
```

Comment:
shouldn't you also check that pnl is correclty calculated? That's the whole point of accounting.